src/App.tsx

```diff
@@ -1,24 +1,95 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Header } from './components/header/Header';
+import { List } from './components/list/List';
+import { Footer } from './components/footer/Footer';
 import { UserWarning } from './UserWarning';
+import { Warnings } from './components/warnings/Warnings';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Select } from './types/Select';
+import { Error } from './types/Error';
 
-const USER_ID = 0;
+const USER_ID = 10514;
+
+const { All, Active, Completed } = Select;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<[] | Todo[]>([]);
```

Comment:
it is enough just use `<Todo[]>`
```suggestion
  const [todoList, setTodoList] = useState<Todo[]>([]);
```

Reply:
fix all such cases with arrays